sycl/source/detail/ur.cpp

```diff
@@ -439,6 +439,98 @@ ur_program_metadata_t mapDeviceBinaryPropertyToProgramMetadata(
 }
 
 } // namespace ur
+
+const char *stringifyErrorCode(int32_t error) {
```

Comment:
Would it be possible to use the existing print api?
https://github.com/oneapi-src/unified-runtime/blob/main/include/ur_print.hpp#L1773